cli/src/main/java/com/okta/cli/commands/Register.java

```diff
@@ -44,6 +44,12 @@
     @CommandLine.Option(names = "--company", description = "Company/organization used when registering a new Okta account.")
     protected String company;
 
+    @CommandLine.Option(names = "--country", description = "Country of residence")
+    protected String country;
+
+    @CommandLine.Option(names = "--oie", description = "Create Okta account with OIE enabled.", hidden = true)
+    protected Boolean oie = null;
```

Comment:
Does this mean that users with a current version of the CLI will be able to create classic orgs?

Reply:
Possibly, (at least for now), there are a few angles we could take here, but ideally, the Okta CLI, needs to start using this newer registration API anyway. So it _might_ be the easiest path forward.

This value will probably default to `true`, I'm not 100% we want to support this flag (though it might be handy)